worker/uniter/runner/runner.go

```diff
@@ -511,6 +511,12 @@ func (b *bufferAdaptor) Messagef(isPrefix bool, message string, args ...interfac
 	b.outCopy.WriteString(formattedMessage)
 }
 
+func (b *bufferAdaptor) Bytes() []byte {
```

Comment:
ideally comment public funcs but i see we're not consistent with that in this file

Reply:
But that doesn't mean we shouldn't in the future :wink: